### PR TITLE
deprecated old methods

### PR DESCRIPTION
**Pull request by Mav7**
@skippyPeanutButter Deprecated ComicsEngine#dbtable_assets_match?, ComicsEngine#add_appendage, and ComicsEngine#remove_appendage; also put every other class under a namespace ```WebComics```. ComicsEngine#add_appendage introduced a hard to diagnose bug so I had to do away with it. This is for version  1.1.0 of the gem. It is still backwards compatible with older versions. I'm following this [guide for versioning](http://semver.org/)

### CODE REVIEW

File: lib/comics_engine.rb [hunk 1]
```diff
@@ -1,24 +1,64 @@
...
     @comics_dir = ComicsDir.new(@assets)
   end
 
   def dbtable_assets_match?
+    warn "[DEPRECATION] `dbtable_assets_match?` is deprecated. Please use `ComicsDB.exists?` instead."
     ComicsDB.exists?(@table_name, @assets)
   end
 
   def add_appendage
+    warn "[DEPRECATION] `add_appendage` is deprecated."
```
Comment (skippyPeanutButter) [inline lib/comics_engine.rb]: In your other warnings there are alternative methods that are listed. Is there one for this one as well?

File: lib/comics_engine.rb [hunk 2]
```diff
@@ -1,24 +1,64 @@
...
     @o_assets = @assets
-    @assets = "#{@assets}/#{@table_name}"
+    @assets = "#{@assets}/#{@table_name}"  
   end
 
   def remove_appendage
+    warn "[DEPRECATION] `remove_appendage` is deprecated."
```
Comment (skippyPeanutButter) [inline lib/comics_engine.rb]: See add_appendage

**Comment by skippyPeanutButter** [comment]
If you look at the travis-ci output, I believe you're still using the deprecated methods in your tests.

File: lib/comics_engine.rb (hunk 1 above)
Comment (Mav7) [inline lib/comics_engine.rb]: No this one is completely taken out. It was the cause of a hard to diagnose bug in CC so it had to go.

**Comment by Mav7** [comment]
@skippyPeanutButter Yeah am I not suppose to?

**Comment by skippyPeanutButter** [comment]
Well I just figured since you're deprecating those methods and replacing their functionality with other code, should you not use that updated code for your tests instead of the deprecated methods?

Your tests should not be static things, meaning if you update your code you should also update your tests as necessary to reflect the changes.

**Comment by Mav7** [comment]
@skippyPeanutButter I see I'll do that next. I thought I'd keep them in there to see if for some reason it breaks something else which it never did anyway we're fine.

**Pull request merged**